x/distribution/client/testutil/suite.go

```diff
@@ -799,7 +799,6 @@ func (s *IntegrationTestSuite) TestNewWithdrawAllTokenizeShareRecordRewardCmd()
 // `NumValidators` the existing tests are leading to non-determnism so created new suite for this test.
 func (s *IntegrationTestSuite) TestNewWithdrawAllRewardsGenerateOnly() {
 	val := s.network.Validators[0]
-	val1 := s.network.Validators[1]
```

Comment:
why remove the extra val and associated tests below?

Reply:
This test was written off of SDK 46 which has 2 vals at startup by default; whereas, SDK 45 has 1 val. I didn't want to add another val to the startup cause it might break other tests, and this is just a CLI test meant to verify the command parses properly, so I don't think it's that necessary to have two different validators